parsec/mca/device/device_gpu.c

```diff
@@ -14,6 +14,9 @@
 #include "parsec/execution_stream.h"
 #include "parsec/utils/argv.h"
 #include "parsec/parsec_internal.h"
+#include "parsec/scheduling.h"
+
+#define PARSEC_CUDA_DATA_COPY_ATOMIC_SENTINEL 1024
```

Comment:
This define moved in the `mca/device/device_gpu.c` but kept it's original name that includes  `CUDA` ?

Reply:
Forgot to rename. See b098633e7